ethpm_cli/registry.py

```diff
@@ -0,0 +1,156 @@
+import json
+from pathlib import Path
+from typing import Any, Dict, Iterable, NamedTuple, Optional, Tuple
+
+from eth_typing import URI
+from eth_utils import to_dict
+from eth_utils.toolz import assoc, assoc_in, dissoc
+from ethpm.backends.registry import is_valid_registry_uri, parse_registry_uri
+
+from ethpm_cli._utils.filesystem import atomic_replace
+from ethpm_cli._utils.logger import cli_logger
+from ethpm_cli.config import Config
+from ethpm_cli.constants import REGISTRY_STORE
+from ethpm_cli.exceptions import InstallError
+
+
+class StoredRegistry(NamedTuple):
+    uri: URI
+    active: bool = False
+    alias: Optional[str] = None
+    ens: Optional[str] = None
+
+    @property
+    def format_for_display(self) -> str:
+        activated = "(active)" if self.active else ""
+        return f"{self.uri} --- {self.alias} {activated}"
```

Comment:
You might include the leading whitespace in the `activated` variable so that there isn't a dangling space when it isn't activated.

```python
activated = " (active)" if self.active else ""
return f"{self.uri} --- {self.alias}{activated}"
```